### PR TITLE
OCM-9205 Setup TF day1 supplemental jobs

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

**Comment by radtriste** [comment]
cc @xueli181114 for approval

File: ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml [hunk 1]
```diff
@@ -273,6 +273,21 @@ tests:
     test:
     - ref: rhcs-e2e-tests
     workflow: rhcs-aws-sts
+- as: rosa-day1-supplemental-f3
```
Comment (xueli181114) [inline ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:276]: No slack message posting?

**Review by ciaranRoche — APPROVED** [review]
/lgtm

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-day1-supplemental-optional-presubmit

**Comment by radtriste** [comment]
/pj-rehearse pull-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-day1-supplemental-optional-presubmit periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-day1-supplemental-f3

**Pull request merged**